src/globalInfo/accessors/sandboxAccessor.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Nullable } from '@salesforce/ts-types';
+import { GlobalInfo } from '../globalInfoConfig';
+import { SfInfoKeys, SfSandbox, SfSandboxes } from '../types';
+
+export class SandboxAccessor {
+  public constructor(private globalInfo: GlobalInfo) {}
+
+  /**
+   * Returns all the sandboxes for all the values
+   */
+  public getAll(): SfSandboxes;
+  /**
+   * Returns all the sandboxes for a given prod org
+   *
+   * @param entity the sandbox entity that you want to get the sandboxes for
+   */
+  // eslint-disable-next-line @typescript-eslint/unified-signatures
+  public getAll(entity: string | SfSandbox): SfSandboxes;
```

Comment:
what makes this necessary?

Reply:
I assume you are referring to the method signature.

This function will return all sandboxes owned by the supplied prod org. 

I need to fix the doc for param entity to make this more clear.